bc_obps/registration/api/user_operator.py

```diff
@@ -61,8 +50,16 @@ def get_user_operator(request, user_operator_id: int):
     return {**user_dict, **operator_dict}
 
 
+@router.get("/operator-has-admin/{operator_id}", response={200: bool, codes_4xx: Message})
+def get_user_operator_admin_exists(request, operator_id: int):
+    hasAdmin = UserOperator.objects.filter(operator_id=operator_id, role=UserOperator.Roles.ADMIN, status=UserOperator.Statuses.APPROVED).exists()
+    return 200, hasAdmin
```

Comment:
nit: Since we are returning multiple responses, we must handle the possibility of an error.: https://django-ninja.dev/guides/response/?h=response#multiple-response-schemas
unless there's no need to cover that error message and our response could just be 200.